projects/fishtornado_xr/Shark.cpp

```diff
@@ -78,22 +78,25 @@ void Shark::Shutdown()
     mMaterialConstants.Destroy();
 }
 
-void Shark::Update(uint32_t frameIndex)
+void Shark::Update(uint32_t frameIndex, uint32_t currentViewIndex)
```

Comment:
As Aliya suggested, if we only Call UpdateTime once, there is no need to check the view index in the Update() function anymore